docs/source/en/index.md

```diff
@@ -249,6 +249,8 @@ Flax), PyTorch, and/or TensorFlow.
 |                     [SegFormer](model_doc/segformer)                     |       ✅        |         ✅         |      ❌      |
 |                           [SEW](model_doc/sew)                           |       ✅        |         ❌         |      ❌      |
 |                         [SEW-D](model_doc/sew-d)                         |       ✅        |         ❌         |      ❌      |
+|                        [SigLIP](model_doc/siglip)                        |       ✅        |         ❌         |      ❌      |
+|            [SiglipVisionModel](model_doc/siglip_vision_model)            |       ✅        |         ❌         |      ❌      |
```

Comment:
I wonder how I make `SiglipVisionModel` to be not included in index.md. I've added it in check_copies.py to not be included in the README but it's still appearing here. `CLIPVisionModel` is also not appearing here.

@ydshieh do you know?